Tools/scripts/run-coverage.sh

```diff
@@ -20,6 +21,21 @@ OPTS="--speedup=$SPEEDUP --timeout=$TIMEOUT --debug --no-clean"
 
 rm -rf build
 
+package_is_installed() {
+    python -m pip --disable-pip-version-check show "$1" | grep -c "Version"
+}
+
+if [ $(package_is_installed "mavproxy") -eq 0 ]; then
+    python -m pip install --user mavproxy
+    python -m pip uninstall -y pymavlink
+fi
+
+
+if [ $(package_is_installed "pymavlink") -eq 0 ]; then
```

Comment:
We've seen this elsewhere and asked for it to be fixed.

Reply:
fixed